tests/integration_tests/test_mongo_task.py

```diff
@@ -490,7 +631,12 @@ def connector(self, integration_mongodb_config):
     def traversal_node(self, example_datasets, integration_mongodb_config):
         dataset = FidesopsDataset(**example_datasets[1])
         graph = convert_dataset_to_graph(dataset, integration_mongodb_config.key)
-        node = Node(graph, graph.collections[0])  # customer collection
+        customer_details_collection = [
```

Comment:
I don't think this will ever be significant since `graph.collections` is limited to the number of collections in the datastore. We could save ourselves some iterations by using:
```
customer_details_collection = None
for collection in graph.collections:
    if collection.name == "customer_details":
        customer_details_collection = collection
        break
```